threescale_api/resources.py

```diff
@@ -1147,7 +1160,12 @@ def service(self) -> 'Service':
         return self.proxy.service
 
 
-class ProviderAccount(DefaultStateResource):
+class ProviderAccount(DefaultResource):
```

Comment:
While not necessary, would it be possible to add tests for this resource?

Reply:
The only test for this resource would be "list" test, as 3scale endpoint: '/provider' is enabled only for list and update. Uptade is not used and probably will not be used in the future.